### PR TITLE
Changing images with code to code snippets

### PR DESCRIPTION
**Pull request by maxagin**
https://user-images.githubusercontent.com/98249521/196821101-3543f493-fa4b-436a-a872-be57503c4c39.mov

### CODE REVIEW

File: extension/resources/walkthrough/plots.md [hunk 1]
```diff
@@ -12,6 +20,27 @@ from the command palette to open it.
 CSV, or TSV files; or save your own plot images (`.png`, etc.). If you're using
 Python, the [DVCLive] helper library can save plots data for you!
 
+```yaml
```
Comment (sroy3) [inline extension/resources/walkthrough/plots.md:23]: These are all python snippets and not yaml

File: extension/resources/walkthrough/setup-project.md [hunk 2]
```diff
@@ -14,25 +14,60 @@ in a [Terminal](command:workbench.action.terminal.new). It will generate a
...
 </p>
+-->
 
 💡 Names, values in this file are project dependent and can be customized.
 
 DVC and this extension read experiments data from these files (e.g
 `metrics.json`, `params.yaml`, etc). Your code needs to write and read to them
 (the example below is Python, but it can be done in any language):
 
+```yaml
```
Comment (sroy3) [inline extension/resources/walkthrough/setup-project.md:44]: This snippet and the next one are also python snippets.

**Review by sroy3 — CHANGES_REQUESTED** [review]

**Review by shcheklein — CHANGES_REQUESTED** [review]
This time we have two PR in one (unrelated changes). Also, go ahead and remove the images and commented code.

File: extension/resources/walkthrough/plots.md [hunk 3]
```diff
@@ -12,14 +20,25 @@ from the command palette to open it.
...
+with open("plots.json", "w") as fd:
+     json.dump({"prc": [
+          {"precision": p, "recall": r, "threshold": t}
+               for p, r, t in points
+     ]})
+```
+
+```python
+from matplotlib import pyplot as plt fig, axes = plt.subplots(dpi=100)
+---
```
Comment (sroy3) [inline extension/resources/walkthrough/plots.md:34]: This look like a merge conflict indicator. Is that wanted?

**Comment by maxagin** [comment]
> This time we have two PR in one (unrelated changes). 

How can I fix this please

**Review by shcheklein — COMMENTED** [review]
LGTM. Can you please show the video demo as we usually do in the description for the other PRs.

File: extension/resources/walkthrough/setup-project.md [hunk 4]
```diff
@@ -14,25 +14,40 @@ in a [Terminal](command:workbench.action.terminal.new). It will generate a
...
+  deps:
+    - data/features
+    - src/train.py
+  params:
+    - epochs
+  outs:
+    - models/predict.h5
+  metrics:
+    - metrics.json:
+        cash: false
```
[the remark this answers is not in the document]
Comment (maxagin) [reply extension/resources/walkthrough/setup-project.md]: You right. Fixing it now. Thank you @sroy3

File: extension/resources/walkthrough/setup-project.md [hunk 5]
```diff
@@ -14,25 +14,40 @@ in a [Terminal](command:workbench.action.terminal.new). It will generate a
...
+  deps:
+    - data/features
+    - src/train.py
+  params:
+    - epochs
+  outs:
+    - models/predict.h5
+  metrics:
+    - metrics.json:
+        cashe: false
```
Comment (sroy3) [inline extension/resources/walkthrough/setup-project.md:29]: There is still a typo: `cashe` should be `cache`

**Review by sroy3 — APPROVED** [review]

**Comment by julieg18** [comment]
Is there a reason we haven't merged this yet? Looks like it has three approvals. cc @maxagin

**Review by shcheklein — APPROVED** [review]

**Comment by shcheklein** [comment]
Thanks @maxagin , good stuff, let's merge it!

**Pull request merged**